frontend/src/metabase/components/CollectionLanding.jsx

```diff
@@ -197,6 +197,18 @@ class DefaultLanding extends React.Component {
     const collectionHasPins = pinned.length > 0;
     const collectionHasItems = unpinned.length > 0;
 
+    let showSidebar = false;
```

Comment:
Generally I'd prefer 

```javascript
const showSidebar = 
  // if the user has write permissions or if there are collections then show the sidebar
  (collection.can_write || collections.length > 0) &&
  // there should also be at least one item, otherwise we have a different
  // new collection CTA
  collectionHasItems;
```

but not a big deal.